membership/src/main/scala/zio/membership/hyparview/HyParView.scala

```diff
@@ -0,0 +1,97 @@
+package zio.membership.hyparview
+
+import zio._
+import zio.stm._
+import zio.membership.transport.{ ChunkConnection, Transport }
+import zio.random.Random
+import zio.macros.delegate._
+import com.github.ghik.silencer.silent
+import zio.console.Console
+import zio.membership.Membership
+import zio.membership.ByteCodec
+import zio.duration._
+import zio.clock.Clock
+
+object HyParView {
+
+  @silent("deprecated")
```

Comment:
What's deprecated?

Reply:
Stream in 2.13. I've separated that stuff out